.github/workflows/workflow.yml

```
@@ -248,7 +271,50 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0 # Fetch all the tags
+      - name: Get version
+        run: |
+          chcp 65001 #set code page to utf-8
```

I don't understand why this is necessary 🤔 

```suggestion
          chcp 65001 # Set encoding to utf-8
```